chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -9,6 +9,537 @@
+      "policy": "custom"
+    },
+    {
+      "name": "grant.rewards.brave.com",
+      "policy": "custom",
+      "mode": "force-https"
+    },
+    {
+      "name": "grant.rewards.bravesoftware.com",
+      "policy": "custom",
+      "mode": "force-https"
+    },
+    {
+      "name": "grant.rewards.brave.software",
+      "policy": "custom",
+      "mode": "force-https"
+    },
+    {
+      "name": "mywallet.ads.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "mywallet.ads.bravesoftware.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "payment.rewards.brave.com",
+      "policy": "custom",
+      "mode": "force-https"
+    },
+    {
+      "name": "payment.rewards.bravesoftware.com",
+      "policy": "custom",
+      "mode": "force-https"
+    },
+    {
+      "name": "payment.rewards.brave.software",
+      "policy": "custom",
+      "mode": "force-https"
+    },
+    {
+      "name": "rewards.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "search.anonymous.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "search.anonymous.bravesoftware.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "static.ads.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "static.ads.bravesoftware.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+
+    // Search
+    {
+      "name": "search.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "cdn.search.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "fg.search.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "imgs.search.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "tiles.search.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "collector.wdp.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "patterns.wdp.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "quorum.wdp.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "star.wdp.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+
+    // Premium
+    {
+      "name": "account.brave.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "account.bravesoftware.com",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+    {
+      "name": "account.brave.software",
+      "mode": "force-https",
+      "policy": "custom"
+    },
+
+    // Test page using a CA outside of the pinset (expected to be blocked)
```

Comment:
This entry can be removed. It's only there to test pinning. It doesn't need to be in our HSTS list.

Reply:
Fixed in https://github.com/brave/brave-core/pull/17033/commits/84b00b9a256cd4411301086df7f38e863a0cd082, thanks!